packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts

```diff
@@ -32,11 +33,13 @@ export function supportChangeEtherBalances(Assertion: Chai.AssertionStatic) {
             actualChanges.forEach((change, i) => {
               if (!change.eq(BigNumber.from(balanceChanges[i]))) {
                 lines.push(
-                  `Expected ${
+                  `Expected ether balance of ${
```

Comment:
Shouldn't this be _the_ ether balance, like in `changeEtherBalance`?

Reply:
Great catch, thank you. Addressed in https://github.com/NomicFoundation/hardhat/pull/2893/commits/b7051adf1189374e1d21605e6a7c42d46f78c887